symfony/src/Service/GoogleGroupsService.php

```diff
@@ -141,6 +141,19 @@ public function getMembers(string $groupEmail)
         }
     }
 
+    public function getUser(string $user)
+    {
+        try {
+            $result = $this->callApi('users', 'get', [$user]);
+            if ($result) {
+                return $result;
+            }
+            return [];
+        } catch (GoogleException $e) {
+            return [];
```

Comment:
Would it make more sense to return null instead of an empty array, since this is getting a single user object, if it exists?